recipes/subby/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "subby" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a560a838d19749b68dc2d3867eb58ea47fd36f5f69e827f0aa961c9771e89926
+
+build:
+  number: 0
```

Comment:
can this be `noarch: python`?

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Yes. Will do.